### ISSUE TITLE
[Improvement] Existing VPC instructions

### ISSUE THREAD

**Issue by chippyash** [sosedoff/wireguard-aws-gateway#2]
Can you please add instructions on what files (and where) to change to launch this into an existing VPC. I think you have to do it in a couple of places, but would be better to be certain.

Thanks

**Comment by sosedoff** [comment]
Plugging into existing VPC could be tricky, everyone has their own setup so i intentionally didnt include an example.
I can try to provide some instruction, but i honestly dont know what kind of network configuration to target, do you have any examples? Maybe you can share your configs?

**Comment by chippyash** [comment]
Hi. I don't think I ever got to resolving this as I couldn't figure out what to change sufficiently, not being a terraform expert (I use serverless).  However, being able to launch into an existing VPC, is I think, crucial to making this implementation useful to many more people.  I'll try to give it some more time.

**Comment by daz10000** [comment]
Yes, just realized after successfully getting through the init, that I now have two vpcs :( - the new one and the one I wanted to attach my vpn gateway to. I had another cloudformation stack that also wanted to make the VPCs,  so this has to deploy into a separate one.  Otherwise, smooth sailing.  Did you ever sort this out @chippyash ?
